dockerfiles/Dockerfile.openvino-rhel8

```diff
@@ -0,0 +1,102 @@
+# Build stage
+FROM registry.access.redhat.com/ubi8/ubi:8.4
```

Comment:
According to the discussions in https://github.com/pypa/manylinux/issues/1282, I don't think taking dependency on UBI8 is good. Especially, licenses. 
When you add extra yum repos to this dockerfile, it makes the built images non-distributable.  See: "https://developers.redhat.com/articles/ubi-faq#ubi_details" . "Can I add non-UBI RPMs to a UBI image and still redistribute the resultant container image on a non-Red Hat platform?" If I understand correctly, the answer is no. And you added CentOS 8 stream packages to this image too. Then why don't you use a centos 8 stream base image instead? 

Red hat people says:"CentOS Stream 8 is the representation of RHEL 8 Nightly". We don't use nightly packages in productions. Therefore I suggest we should avoid using CentOS Stream 8 too.

Reply:
Do you have any suggestions what we can use

Reply:
What's the purpose of this docker file? Files in this dir are not for build pipelines. We put pipelines' docker files in https://github.com/microsoft/onnxruntime/blob/main/tools/ci_build/github/linux/docker. So, if the packages built from this docker file won't be distributed separately, you may just ubuntu. Any Ubuntu version is fine. If you want to continue to use ubi8, you should avoid using packages from CentOS 8 stream and be careful on license restrictions.

Reply:
This is not part of pipeline but standalone script we maintain to ensure we can build RHEL Docker Image for some customers . We have removed centos8 elements , is this ok to go now.